std_msgs/String color
BlobInfo[] blobs
